### PR TITLE
Tech stack updates

### LINKED ISSUE #2: Updates on Tech-stack used
It seems like the dimensions of the TextView are hard-coded
I highly recommend the use of viewBinding instead of the old findViewById 😂
Something else, I think working directly from the XML code creates a cleaner code than when working sorely from the design.

### PR DESCRIPTION
**Pull request by Ericgacoki**
- [x] This fixes #2 

### More suggestions
- Be sure to unify the app, i.e, create a uniform theme so that when users navigate, they won't feel like they've opened another app.
- Update other `XML` files to us the recommended guidelines. For instance in dimension units.
- If an app is in Kotlin, I don't find it necessary to use Java syntax especially in getters and setters.

overall rating :star: :star: :star: /5

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -31,6 +31,9 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    buildFeatures{
+        viewBinding true
+    }
```
Comment (Ericgacoki) [inline app/build.gradle:36]: can also use **dataBinding**

**Comment by Ericgacoki** [comment]
Hey @Jeffkent01coder, could you please let me know the status of this PR!

**Comment by Jeffkent01coder** [comment]
hello @!Erick i will look for you since am stuck on how to get it back to my github

**Comment by Ericgacoki** [comment]
> hello @!Erick i will look for you since am stuck on how to get it back to my github

Since the PR has no conflicts with your main branch, you can proceed to *Merge* and everything will be fine!
(I'd advise you to go through the changes before merging)

**Comment by Ericgacoki** [comment]
Good work @Jeffkent01coder. 😊

**Comment by Jeffkent01coder** [comment]
Thanks mentor  @Ericgacoki

**Pull request merged**